src/server/helpers/search.js

```diff
@@ -71,11 +72,26 @@ function _fetchEntityModelsForESResults(results) {
 
 	return Promise.map(results.hits, (hit) => {
 		const entityStub = hit._source;
-		const model = utils.getEntityModelByType(entityStub.type);
 
+		if (entityStub.type === 'Area') {
+			return Area.forge({bbid: entityStub.bbid})
```

Comment:
No need for the where clause here - just move the `{gid: entityStub.bbid}` into the forge function call instead of `{bbid: entityStub.bbid}`.